### PR TITLE
Add Events documentation with signed url example on POST resolve

### PR DESCRIPTION
**Pull request by StevenRenaux**
| Q | A
| --- | ---
| Branch | 2.x 
| Doc | Events

Because when the image filtered, it putted as a public resource. I needed to update my acl config and create a listener to solve it. 
I just wanted to share my solution because it was not easy to find in the original documentation.

### CODE REVIEW

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -1,50 +1,51 @@
-
-
-LiipImagineBundle
-=================
```
Comment (dbu) [inline Resources/doc/index.rst:4]: what happened here? is it line endings?

File: Resources/doc/events.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
+
+
+Events
+============
```
Comment (dbu) [inline Resources/doc/events.rst]: please align the number of `=` (and `-` below) to the length of the titles

File: Resources/doc/events.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
+
+
+Events
+============
+
+Events availables in the bundle are **PRE_RESOLVE** and **POST_RESOLVE**.
```
Comment (dbu) [inline Resources/doc/events.rst]:
```suggestion
Events availables in the bundle are ``PRE_RESOLVE`` and ``POST_RESOLVE``.

```

File: Resources/doc/events.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
+
+
+Events
+============
+
+Events availables in the bundle are **PRE_RESOLVE** and **POST_RESOLVE**.
+Both receive a CacheResolveEvent as event .
```
Comment (dbu) [inline Resources/doc/events.rst]:
```suggestion
Both receive a CacheResolveEvent as event.

```

File: Resources/doc/events.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+
+Called before url to the cached filtered image is generated.
+
+
+POST_RESOLVE
+-----------------
+
+Called after url to the cached filtered image is generated.
+
+Here is an implementation example about users media with differents filters and an S3 as IONOS. At the end we will update the url to get a temporary signed url to a private resource:
```
Comment (dbu) [inline Resources/doc/events.rst:22]:
```suggestion
Example: Signed URLs
--------------------

Here is an implementation example about users media with differents filters and an S3 as IONOS. At the end we will update the url to get a temporary signed url to a private resource:

```

File: Resources/doc/cache-resolver/aws_s3.rst [hunk 6]
```diff
@@ -263,7 +263,6 @@ for the list of available options.
 Note, that the following options are configured automatically and will be
 ignored, even if you configure it via ObjectOptions:
 
-* ``ACL``
```
Comment (dbu) [inline Resources/doc/cache-resolver/aws_s3.rst:266]: you could add a note to the end of this chapter to reference the example you provide in the events documentation. users might not look at events when they try to figure out how to do signed urls with s3

**Review by dbu — COMMENTED** [review]
thanks, good example of how the event system can be used :+1:

File: Resources/doc/index.rst (hunk 1 above)
Comment (StevenRenaux) [reply Resources/doc/index.rst:4]: It's like the original file, don't know why my commit pushed as a new file. The line 45 is the only one i added.

File: Resources/doc/cache-resolver/aws_s3.rst [hunk 7]
```diff
@@ -307,8 +306,10 @@ You can also use the constructor of the resolver to directly inject multiple opt
                 - { CacheControl: "max-age=86400" }
             tags:
                 - { name: "liip_imagine.cache.resolver", resolver: "aws_s3_resolver" }
+                
+You can find an example with private ACL and signed url on the `events`_ section.
```
Comment (dbu) [inline Resources/doc/cache-resolver/aws_s3.rst:310]: internal links should not be with absolute url, but with the `:doc:` syntax, here 
```
:doc:`events chapter <../events.rst>`
```

**Review by dbu — APPROVED** [review]
thanks a lot, nice example!

i fixed the line endings to be linux endings like the rest of the documentation, and applied the fix for my last comment and merged this in 5bf7e87f841cc6143dc8173b5b06bb12728cde8c

**Pull request closed without merging**